### ISSUE TITLE
Page forbidden

### ISSUE THREAD

**Issue by vovakulikov** [avocode/avocode#1718]
* **Version**: 2.16.2

* **Web/Desktop**: Desktop

* **Platform**: linux

* **Avocode UID**: 208682
---

```
Error: Page forbidden
    at t (file:///home/vovakulikov/avocode/resources/app.asar/dist/avocode.min.js:1:126682)
    at t (file:///home/vovakulikov/avocode/resources/app.asar/dist/avocode.min.js:1:126573)
    at t (file:///home/vovakulikov/avocode/resources/app.asar/dist/avocode.min.js:1:127501)
    at new t (file:///home/vovakulikov/avocode/resources/app.asar/dist/avocode.min.js:1:128088)
    at e.<anonymous> (file:///home/vovakulikov/avocode/resources/app.asar/dist/avocode.min.js:1:1011935)
    at tryCatch (/home/vovakulikov/avocode/resources/app.asar/node_modules/regenerator-runtime/runtime.js:64:40)
    at Generator.invoke [as _invoke] (/home/vovakulikov/avocode/resources/app.asar/node_modules/regenerator-runtime/runtime.js:355:22)
    at Generator.prototype.(anonymous function) [as next] (/home/vovakulikov/avocode/resources/app.asar/node_modules/regenerator-runtime/runtime.js:116:21)
    at r (file:///home/vovakulikov/avocode/resources/app.asar/dist/avocode.min.js:1:1010478)
    
```

**Comment by jakubzitny** [comment]
Hi @vovakulikov, thanks for reporting this error. We already know about it and we have it fixed in our development branch. We'll release the fix next week in `2.16.3`.

**Comment by vojtatranta** [comment]
This should not pop up in the desktop app. What were you trying to do?
